gspread/worksheet.py

```diff
@@ -1157,6 +1162,41 @@ def update_title(self, title):
         self._properties["title"] = title
         return response
 
+    def update_tab_color(self, color):
+        """Changes the worksheet's tab color.
+
+        :param dict color: The red, green and blue values of the color, between 0 and 1.
+        """
+        body = {
+            "requests": [
+                {
+                    "updateSheetProperties": {
+                        "properties": {
+                            "sheetId": self.id,
+                            "tabColor": {
+                                "red": color["red"],
+                                "green": color["green"],
+                                "blue": color["blue"],
+                            },
+                            "tabColorStyle": {
+                                "rgbColor": {
+                                    "red": color["red"],
+                                    "green": color["green"],
+                                    "blue": color["blue"],
+                                }
+                            },
+                        },
+                        "fields": "tabColor,tabColorStyle",
```

Comment:
You said you ran some manual tests and it works ? Last time I tried to do it, here in the `fields`I had to put every single entry that is updated, so `tabColor.red,tabColor.green,tabColor.blue,tabColorStyle.rgbColor.red.....`

If it works likes this then it's better, simpler to read. :slightly_smiling_face: